### DISCUSSION TITLE
新项目提案激励流程及激励规则

### DISCUSSION THREAD

**Discussion by mosquitohater2** [FIBOSIO/fibos#23] [General]
之前【FIBOS 开源委员会】提出了开发者专项扶持计划，委员会收到了很多开发者提出的有趣想法，因此想要在此征集一下大家对于激励流程及激励规则的建议~

**Comment by LEONG-BTC** [comment]
搞起来

**Comment by kilmas** [comment]
1.是先提供资金，还是做出产品再奖励？分配比例？
步骤提交产品资料，使用AMM，demo等

**Comment by kilmas** [comment]
主要激励产品。
1.基础设施，跨链，evm虚拟机。
2.市面defi产品，dex，稳定币swap，借贷，流动性挖矿，杠杆挖矿，杠杆交易等产品。

**Comment by kilmas** [comment]
持续运营激励：按照产品使用人数，提供后续激励

**Comment by BIU64** [comment]
应 FIBOS 社区提案《新项目提案激励流程及激励规则》https://github.com/FIBOSIO/fibos/discussions/23，结合@还是红领巾(Github@kilmas)等开发者委员会成员讨论内容，现提出以下草案规则，望可实现社区治理前提下、公平公开的 FIBOS 社区激励机制：

### 提案目标：
1.解决 FIBOS 生态开发激励机制缺失问题，建立开放机制，吸引独立开发者和其他 DeFi 团队参与 FIBOS 生态，提高已有生态设施的利用率、FO 的应用和沉淀场景。
2.确保激励发放给对生态有价值的项目、确保激励发放过程透明公开公平、确保激励申请开放和公正，根据项目的实际进度发放，根据不同项目实际情况独立设置激励额。

### 提案具体内容：
1.生态激励可分为**基础设施类**和**生态赋能类**。
基础设施类即非盈利性，但社区必要的技术和产品开发。如钱包、投票治理工具、区块链浏览器等的开发和持续维护运营。
生态赋能类即能够在用户、链上资产、FO 的 DeFi 应用等方面做出贡献的项目或事务计划。

2.建立激励的申请核验和拨付机制。
**开发者委员会**：由开发者为主组建开发者委员会(已有)，负责对所有的申请予以技术可行性、技术难度等方面进行审核，筛选出具有讨论意义的项目。
**FO 持有者投票**：由 FO 持有用户投票，决定是否给予拨付。
**BP/基金会**：由 BP 或基金会核验链上投票结果并执行拨付流程。如认为无法拨付，需与委员会成员沟通并向社区公开原因。

3.具体的生态激励流程：
申请期
A.小明认为 FIBOS 社区需要有项目 X，并在社区内与不同开发者或用户有过探讨。
B.小明撰写申请提案，内容包括：提案说明（什么样的产品或事务、对 FIBOS 生态的价值、具体的实现方案、小明的简介）和提案执行规划（时间节点、交付给社区的可验证的成果）
C.小明联系【开发者委员会】，申请讨论

投票期
D.开发者委员会认为方案可行，60% 以上成员在提案贴内明确回复可行后，小明可申请发起投票。
E.小明说明申请额度、拨付方案、投票周期、投票方式
F.【BP/基金会】按照【FO 持有者投票】的结果拨付 FO

拨付期
G.小明按期提交核验资料，得到【开发者委员会】核验通过后申请【BP/基金会】继续拨付
H.小明完成提案项目，交付代码并获得最终部分的 FO 激励

4.其他细节
激励额度：建议按周期设定激励总额，由不同项目自行博弈和争夺。
激励发放：由提案者根据实际需求拟定，但至少分为 3 期，比例不超过 5、3、2，分期支付，以确保激励有效性
关于投票：
A.单一提案投票中 FO 持有者投票额至少超过流通量 5% 才能视为有效投票（50,000,000）
B.投票方式需合理合规无漏洞，否则【BP/基金会】可拒绝拨付
C.投票中明确表达同意的票数超过 51%，该提案才能被认为有效
D.目前尚无有效的社区投票工具，现提出一可用方案：a.根据 FO持有数量（Stake+流动）计算权重 b.任意地址在三天内（解锁期限）向特定地址中转入特定数量的 FO 完成投票 c.当投票期截至时快照，并根据快照时不同地址持有 FO 数量确定票权。
E.如某提案具有可实现属性，但报价过高导致未被投票通过，则可重新申请投票。但其多次重复申请必然会导致【开发者委员会】或 FO 持有用户疲烦，从而形成报价和投票的博弈。

### 总结：
1.整体流程简单，避免过度设计和繁琐，避免对个人或工具的依赖，确保去中心化
2.建立博弈的机制设计，不论激励额度、激励有效性还是方案审核，通过多种方式筛选和确保提案有效性，避免恶意提案或攻击
3.具有可持续性，有利于 FIBOS 逐步形成高效的社区化治理
4.首批建议社区有志开发同学关注包括投票、资产多签拨付、提案追踪和提案内容的链上存证读取、链上数据监控（用户数、资产 TVL 等）等工具
5.最终的 DeFi 核心生态其实很难被激励直接砸出来，个人认为应当优先解决用户入口、吸引用户使用或 FOMO 效应等方面。

本方案仅为个人思路，如有疏漏或不足之处请随时指证。欢迎迭代或修正版本，共同推动社区发展。

**Comment by BIU64** [comment]
提出更新：
4.其他细节部分新增【从交易所地址进行的投票属于无效投票】
